README.md

```diff
@@ -1,4 +1,16 @@
 wp-cli/ideas
 ============
 
-Have an idea for a WP-CLI command? [Create a new issue](https://github.com/wp-cli/ideas/issues/new) or [up-vote an existing issue](https://github.com/wp-cli/ideas/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc).
+### Have an idea for a WP-CLI command?**
+
+Feel free to [up-vote an existing issue](https://github.com/wp-cli/ideas/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc) if you can find one that matches, or [create a new issue](https://github.com/wp-cli/ideas/issues/new) if no one has proposed the idea yet.
```

Comment:
> What does "up-voting" actually mean?

I've changed the language to "use an emoji reaction".

Up-voting probably too strongly implies new feature creation is a purely democratic process (e.g. the most up-votes gets picked). Emoji reactions are better used to gauge level of interest.